apps/web/.env.example

```diff
@@ -1,4 +1,6 @@
+NEXTAUTH_SECRET= # Linux: `openssl rand -hex 32` or go to https://generate-secret.now.sh/32
 TWITTER_BEARER_TOKEN=
 TWITTER_CLIENT_ID=
 TWITTER_CLIENT_SECRET=
-NEXTAUTH_SECRET= # Linux: `openssl rand -hex 32` or go to https://generate-secret.now.sh/32
\ No newline at end of file
+TWITTER_PROFILE_ID=#2313873457
+TWITTER_PROFILE_USER=#sseraphini
```

Comment:
why twitter profile id and user is required?

Reply:
cc sseraphini as a service

Reply:
sounds good, we need to document this somehow

Reply:
I think we can avoid repeat your username in the code, so if in the future we build that for any people runs your instance, we'll not need to do refactor all the codebase.
See:
![image](https://user-images.githubusercontent.com/5343791/159576633-ede3098d-75f4-44a1-92b8-02f55906fe1f.png)